R4ML/R/util.io.R

```diff
@@ -83,6 +83,38 @@ r4ml.warn <- r4ml.gen.logger("WARN")
 r4ml.err <- r4ml.gen.logger("ERROR")
 r4ml.fatal <- r4ml.gen.logger("FATAL")
 
+#' r4ml.setLogLevel
+#' @description Set log level for the current session.
+#' @param level the level to be used. The various log levels that are supported are "ALL", "TRACE",
+#' "DEBUG", "INFO", "WARN", "ERROR", "FATAL", "OFF" in the increasing priority.
+#' @param set.java.log logical variable to set java log level. Default is FALSE.
+#' @details When R4ML session is started, the defalult log level is set to "INFO". This function allows to
+#' set the desired log level after the session is started. The java log level used by SystemML is set to ERROR
+#' by default. That behaviour can also be changed by setting the set.java.log flag to TRUE.
+#' @usage r4ml.setLogLevel("WARN")
+#' @examples \dontrun{
+#' r4ml.session()
+#' r4ml.setLogLevel("WARN")
```

Comment:
usually R follow convention of one dot but we have two dot in set.java.log , I was thinking if we can come up with one dot and some reasonable name? or other option ? java.log sounds reasonale. since the name of func is setLogLevel user knows it is set so no need for putting set in set.java.log